# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-contain/contain-body-t-o-004.html.ini
# Commit: 86e9449aba35
# Full Hash: 86e9449aba35c7439e8a41a2804388994462c00d
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r c7d1d0cfaa6c -r 86e9449aba35 testing/web-platform/meta/css/css-contain/contain-body-t-o-004.html.ini
--- a/testing/web-platform/meta/css/css-contain/contain-body-t-o-004.html.ini	Wed Jun 22 10:30:03 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[contain-body-t-o-004.html]
-  expected: FAIL